influxql/ast.go

```diff
@@ -3630,7 +3630,7 @@ func TimeRange(expr Expr) (min, max time.Time, err error) {
 }
 
 // TimeRangeAsEpochNano returns the minimum and maximum times, as epoch nano, specified by
-// and expression. If there is no lower bound, the start of the epoch is returned
+// and expression. If there is no lower bound, the minimum time is returned
```

Comment:
Good catch. I went through a bunch of different revisions while doing this and I might have lost some things like this.